### PR TITLE
Token coordinates

### LINKED ISSUE #1735: "Create related card" places tokens incorrectly
Tokens created by this function should be placed onto the board using the same rules the `create token` dialog uses:

* Use the tablerow information associated with the card
* Don't place the token in a group of cards

---

Currently, the tablerow information is ignored; all the cards are being created in the top row.
If the card producing tokens is in the first position on the battlefield (top row, far left), the related tokens are placed on top of it, instead of in a group of like-cards.

Can more code be re-used from the create-token dialog?

### PR DESCRIPTION
**Pull request by ZeldaZach**
Fix #1735

This sets the x&y coordinates of the token so they don't bunch up and they take free space only.

Before:
screenshot 2015-12-29 23 41 36

After:
screenshot 2015-12-29 23 41 19

### CODE REVIEW

**Comment by tooomm** [comment]
> Currently, the tablerow information is ignored; all the cards are being created in the top row.

Is this issue from #1735 also addressed?

**Comment by ctrlaltca** [comment]
Yes, the tablerow information is fetched from the card (token).

File: cockatrice/src/player.cpp [hunk 1]
```diff
@@ -1136,6 +1136,9 @@ void Player::actCreateRelatedCard()
     // get the target card name
     QAction *action = static_cast<QAction *>(sender());
     CardInfo *cardInfo = db->getCard(action->text());
+    
+    // get the target token's location
+    QPoint gridPoint = QPoint(-1, table->clampValidTableRow(2 - cardInfo->getTableRow()));
```
Comment (Daenyth) [inline cockatrice/src/player.cpp]: This seems worth wrapping in a function with a comment explaining what this is doing, and why `-1` and `2` are used.

**Comment by Daenyth** [comment]
:+1: and can :shipit: after my comment is addressed.

**Pull request merged**